src/main/java/seedu/foodrem/logic/commands/itemcommands/NewCommand.java

```diff
@@ -8,14 +8,12 @@
 import seedu.foodrem.logic.commands.exceptions.CommandException;
 import seedu.foodrem.model.Model;
 import seedu.foodrem.model.item.Item;
+import seedu.foodrem.viewmodels.ItemWithMessage;
 
 /**
  * Creates a new an item in FoodRem.
  */
 public class NewCommand extends Command {
-    public static final String MESSAGE_SUCCESS = "New item added:\n%1$s";
-    public static final String MESSAGE_DUPLICATE_ITEM = "This item already exists in FoodRem";
```

Comment:
Same here, why was this constant removed?

Reply:
Ditto, answered above. They're only used once.